src/configuration.ts

```diff
@@ -740,7 +747,23 @@ export function getCommandForConfiguration(configuration: string | undefined): v
         logger.message(`Deduced command '${configurationMakeCommand} ${configurationMakeArgs.join(" ")}' for configuration "${configuration}"`);
     }
 
-    // Validation and warnings about properly defining the makefile and make tool.
+    // Check for makefile path on disk: we search first for any makefile specified via the makefilePath setting,
```

Comment:
This whole green block I moved from below, it was by mistake under an IF about not having a build log provided. Having a build log given via "makefile.buildLog" only bypasses the "make" invocation when we calculate IntelliSense but when users want to build, we still need to know where make and makefile are.